src/UI/state/mapSlice.ts

```diff
@@ -45,6 +53,21 @@ export const mapSlice = createSlice({
   name: 'map',
   initialState,
   reducers: {
+    drawerToggle(state) {
+      const map_drawer = state.map_drawer;
+      if (state.map_drawer.open === true) {
+        map_drawer.open = false;
+        map_drawer.overlays = false;
+        map_drawer.baseMap = false;
+      }
+      else{

```

Comment:
Looks like this stuff isn't being picked up by the linter... I'll make a separate PR to fix this